spark/spark-base/src/main/java/com/thoughtworks/go/spark/Routes.java

```diff
@@ -836,4 +836,8 @@ public static String self(long id) {
     public class StatusReports {
         public static final String SPA_BASE = "/admin/status_reports";
     }
+
+    public static class ApiInfo {
+        public static final String BASE = "/api/apis";
```

Comment:
should it be under internal namespace??

Reply:
Yes, we can put it under internal api. //cc @rajiesh